ivykis 0.41 (new formula)

MrAnno opened the pull request:
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

woodruffw commented on Formula/ivykis.rb:
```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
+  url "https://github.com/buytenh/ivykis/archive/v0.41.tar.gz"
```
Let's fetch it from here instead: https://sourceforge.net/projects/libivykis/files/0.41/

woodruffw commented on Formula/ivykis.rb:
```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
```
That page leads to this, which looks like the real homepage: https://sourceforge.net/projects/libivykis/

woodruffw commented:
Debian et al. provide this, so I think we can ignore the notability warning here. It'll also disappear once the `url` link is moved to sourceforge.

MrAnno commented:
Thanks for the review.

I've updated the PR based on your notes. I had some issues with the sha256 sum because of the SourceForge mirrors, that was the reason I switched to the GitHub URL.

woodruffw commented:
No problem!

> I had some issues with the sha256 sum because of the SourceForge mirrors, that was the reason I switched to the GitHub URL.

Could you elaborate? Was the `sha256` varying across mirrors?

MrAnno commented:
> Could you elaborate? Was the sha256 varying across mirrors?

Sorry, I was wrong. All the available SourceForge mirrors have the same checksum, I just mixed up the GitHub and the SourceForge tarball, so I entered a wrong sha256 value.

woodruffw commented:
Thanks @MrAnno, and congratulations on your first contribution/formula! :tada:
